### DISCUSSION TITLE
Component being rendered twice

### DISCUSSION THREAD

**Discussion by praveshishere** [nuxt/nuxt#16971] [Questions]
I have got a breadcrumbs component, to appear at the top of my page.

- When I load the page, the server response is fine, but after a while, two breadcrumbs appear.
- Also, the whole page is being rendered twice, one below the other
- Dev-tools show that one of the versions is actually a node of Vue tree and the other is simple HTML.
which makes me think that hydration is simply not deleting the HTML version, and pushing the interactive node again. which is also evident from the fact that one of the versions is not interactive while the other one is.

Does anyone have any idea why is this happening or how to resolve this?

**Comment by viii47** [comment]
Add a `console.log()` to the component's setup area and see if you see the response multiple times. Would also be helpful to see your breadcumb components markup code.

**Reply by praveshishere** [reply]
> Add a `console.log()` to the component's setup area and see if you see the response multiple times. Would also be helpful to see your breadcumb components markup code.

Console, outputs in both server side and client side once, which is expected

Also, I just have noticed that the whole page is appearing twice consecutively just below the first render

**Comment by danielroe** [comment]
This sounds like a hydration error. Check your console logs in dev mode.

**Reply by praveshishere** [reply]
The following warning does appear but I have got no idea, how is that supposed to be resolved
```js
 [Vue warn]: Hydration node mismatch:
- Client vnode: div 
- Server rendered DOM: <!---->  
  at <EventView> 
  at <[id] onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< undefined > > 
  at <Anonymous key="langCode-event-view-id" routeProps= {Component: {…}, route: {…}} pageKey="langCode-event-view-id" > 
  at <BaseTransition mode="out-in" appear=false persisted=false  ... > 
  at <Transition name="page" mode="out-in" > 
  at <RouterView name=undefined route=undefined > 
  at <NuxtPage page-key="langCode-event-view-id" > 
  at <BaseAppShell class="font-roboto" > 
  at <AsyncComponentWrapper class="font-roboto" > 
  at <BaseTransition mode="out-in" appear=false persisted=false  ... > 
  at <Transition name="layout" mode="out-in" class="font-roboto" > 
  at <Anonymous name="base-app-shell" class="font-roboto" > 
  at <App key=1 > 
  at <NuxtRoot>
```

**Reply by danielroe** [reply]
You can provide a minimal reproduction and see if someone can help you.
